packages/api-plugin-carts/src/resolvers/Subscription/anonymousCartUpdate.js

```diff
@@ -0,0 +1,41 @@
+import ReactionError from "@reactioncommerce/reaction-error";
+import hashToken from "@reactioncommerce/api-utils/hashToken.js";
+import { withFilter } from "graphql-subscriptions";
+
+/**
+ * @summary Filters the subscription result
+ * @param {Object} payload - The subscription payload
+ * @param {Object} variables - The subscription variables
+ * @returns {Boolean} - Whether the subscription result should be sent to the client
+ */
+export function filter(payload, variables) {
+  const { anonymousCartUpdate: cart } = payload;
+  if (!cart) return false;
+
+  const { cartId } = variables.input;
+  return cart._id === cartId;
+}
+
+/**
+ * @summary Publishes the updated cart to the client
+ * @param {Object} _ unused
+ * @param {Object} args - The arguments passed to the subscription
+ * @param {Object} context - The application context
+ * @returns {Promise<Object>} the filtered subscription result
+ */
+async function anonymousCartUpdate(_, args, context) {
```

Comment:
Since this is the subscribe function/query, which is related to an event, I think using a past-tense convention similar to the event name makes sense. In other words, I would add a "d" to make this `anonymousCartUpdated`, and change the file name and schema to match. Same for `accountCartUpdated`.